portfolio/src/main/webapp/index.html

```diff
@@ -54,5 +56,232 @@
+			</div>
+		</div>
+		<div class="row Tutor">
+			<div class="col text">
+				<center><img class="logo" src="Images/CU2.png" alt=""></center>
+			</div>
+			<div class="col leftText">
+				<p>Claflin University</p>
+				<p>2020 - current</p>
+				<p>Tutor</p>
+			</div>
+			<div class="col rightText">
+				<p>I tutor mathematics and Computer Science courses - Introduction to Java. </p>
+			</div>
+		</div>
+		<div class="row NSBE">
+			<div class="col">
+				<center><img class="logo" src="Images/NSBE.png" alt=""></center>
+			</div>
+			<div class="col leftText">
+				<p>National Society of Black Engineers</p>
+				<p>2020 - current</p>
+				<p>Volunteer</p>
+			</div>
+			<div class="col rightText">
+				<p>I am responsible for budgeting and seeking funds for the society</p>
+			</div>
+		</div>
+	</div>
+	</div>
+</section>
+<!--Skills section-->
+<div class="classIntro">
+	<center>
+		<p> Skills </p>
+	</center>
+</div>
+<section id="Skills" name = "next" class="top-padding t">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-12 col-sm-12 a"  >
+            <div class="skills-thumb">
+              <div class="skills_intro">
+              <p >Languages</p>
+              </div>
+              <div class="skills_items">
+                <p>HTML</p>
+                <p>CSS</p>
+                <p>Python</p>
+                <p>Java</p>
+                <p>SQL</p>
+                <p>Javascript</p>
+              </div><br>
+              <div class="skills_intro">
+              <p>Tools</p>
+              </div>
+              <div class="skills_items">
+                <p>Android Studio</p>
+                <p>Git</p>
+                <p>Heroku</p>
+                <p>Postgress</p>
+                <p>Django</p>
+                <p>Flask</p>
+                <p>Selenium</p>
+                <p>Dash</p>
+                <p>Google Firebase</p>
+                <p>Azure</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+</section>
+<!--This is a section for social buttons/connect--->
+<div class="classIntro">
+	<center>
+		<p> Contact? </p>
+	</center>
+</div>
+<section id="footer" class="top-padding">
+	<div class="social-buttons">
+		<a href="https://twitter.com/joshua_kiprono" class="social-buttons__button social-button social-button--twitter" aria-label="Twitter"> <span class="social-button__inner">
+              <i class="fab fa-twitter"></i>
+             </span> </a>
+		<a href="https://www.linkedin.com/in/joshuakiprono/" class="social-buttons__button social-button social-button--linkedin" aria-label="LinkedIn"> <span class="social-button__inner">
+              <i class="fab fa-linkedin-in"></i>
+             </span> </a>
+		<a href="jkiprono@claflin.edu" class="social-buttons__button social-button social-button--mail" aria-label="Mail"> <span class="social-button__inner">
```

Comment:
If you want to put a mail link, the href attribute should look like "mailto:jkiprono@claflin.edu". Although mailto links can be annoying for users who don't have a default mail client set up and don't expect a link to open a program on their computer. I would recommend simply leaving it as plain text.

Reply:
Got it